Add explanation to court phone numbers and changed styling

DeanBullockCGI opened the pull request:
1 migration included.

Tests included.

Updated Ingest file.

This PR is part 1 of a 2 part change for CF-30. Part 2: Admin functionality is WIP.

richerry commented on data/test_data/courts.json:
```diff
@@ -81,27 +81,32 @@
             {
                 "sort": 0,
                 "name": "Enquiries",
-                "number": "01254  687500"
```
So I discovered how to run the manage command to ingest data/court.json:

```
(courtfinder-search) (jessie-py2)richerry@debian:~/Clients/CCS/courtfinder-search$ python courtfinder/manage.py populate-db --ingest
{"timestamp": "2017-02-20 14:34:53,935","name": "courtfinder-search::populate-db.py:", "level": "INFO", "level_no": 20, "message": "handle: Ingesting files..."}
{"timestamp": "2017-02-20 14:34:53,935","name": "courtfinder-search::populate-db.py:", "level": "INFO", "level_no": 20, "message": "import_files: Importing file data/courts.json"}
Traceback (most recent call last):

<SNIP>

  File "/home/richerry/Clients/CCS/courtfinder-search/courtfinder/search/ingest.py", line 66, in courts
    hide_aols=court_obj['hide_aols']
KeyError: 'hide_aols'
```

So I think we need to update this file with the output from the admin side when you have finished the work there and rebased to include the hide_aol work. Do you agree? Shall I hold off merging this PR until the admin work is finished and you can easily include a valid courts.json file?

DeanBullockCGI commented:
You can generate a new courts.json file with the export script giving you the correct structure. The admin side has been completed for this change.

Hope this helps,
Dean

Dean Bullock, Software Engineer - CGI
07919 573 981
Dean.Bullock@cgi.com
________________________________
From: richerry<mailto:notifications@github.com>
Sent: ‎20/‎02/‎2017 14:53
To: ministryofjustice/courtfinder-search<mailto:courtfinder-search@noreply.github.com>
Cc: Bullock, Dean<mailto:dean.bullock@cgi.com>; Author<mailto:author@noreply.github.com>
Subject: Re: [ministryofjustice/courtfinder-search] Add explanation to court phone numbers and changed styling (#161)


@richerry approved this pull request.

________________________________

In data/test_data/courts.json<https://github.com/ministryofjustice/courtfinder-search/pull/161#discussion_r102033183>:

>              },
             {
                 "sort": 0,
                 "name": "Enquiries",
-                "number": "01254  687500"


So I discovered how to run the manage command to ingest data/court.json:

(courtfinder-search) (jessie-py2)richerry@debian:~/Clients/CCS/courtfinder-search$ python courtfinder/manage.py populate-db --ingest
{"timestamp": "2017-02-20 14:34:53,935","name": "courtfinder-search::populate-db.py:", "level": "INFO", "level_no": 20, "message": "handle: Ingesting files..."}
{"timestamp": "2017-02-20 14:34:53,935","name": "courtfinder-search::populate-db.py:", "level": "INFO", "level_no": 20, "message": "import_files: Importing file data/courts.json"}
Traceback (most recent call last):

<SNIP>

  File "/home/richerry/Clients/CCS/courtfinder-search/courtfinder/search/ingest.py", line 66, in courts
    hide_aols=court_obj['hide_aols']
KeyError: 'hide_aols'


So I think we need to update this file with the output from the admin side when you have finished the work there and rebased to include the hide_aol work. Do you agree? Shall I hold off merging this PR until the admin work is finished and you can easily include a valid courts.json file?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ministryofjustice/courtfinder-search/pull/161#pullrequestreview-22776251>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AWduN_IJKDXSPEGNtReOk8NMhkcmjCivks5reajDgaJpZM4L-JVR>.
